codebuild_specs/cleanup_e2e_resources.yml

```diff
@@ -0,0 +1,14 @@
+version: 0.2
+env:
+  shell: bash
+  variables:
+    AMPLIFY_DIR: /root/.npm-global/lib/node_modules/@aws-amplify/cli-internal/bin
+    AMPLIFY_PATH: /root/.npm-global/lib/node_modules/@aws-amplify/cli-internal/bin/amplify
+    CI: true
+    CIRCLECI: true
```

Comment:
think we can get rid of this variable?

Reply:
This will be removed in the next PR of cleanup scripts, which is WIP